test/conformance/enqueue/urEnqueueKernelLaunch.cpp

```diff
@@ -139,8 +139,9 @@ TEST_P(urEnqueueKernelLaunchTest, InvalidKernelArgs) {
                                      nullptr));
 
     if (backend == UR_PLATFORM_BACKEND_CUDA ||
-        backend == UR_PLATFORM_BACKEND_HIP) {
-        GTEST_FAIL() << "AMD and Nvidia can't check kernel arguments.";
+        backend == UR_PLATFORM_BACKEND_HIP ||
+        backend == UR_PLATFORM_BACKEND_LEVEL_ZERO) {
+        GTEST_FAIL() << "AMD, L0 and Nvidia can't check kernel arguments.";
```

Comment:
sooo... this is a native cpu adapter test then?:D

Reply:
Yeah, I guess. Tbh, I don't know if there are any plans to add support for that for other backends.